
From: James Prestwood <prestwoj@gmail.com>
To: iwd@lists.linux.dev
Cc: James Prestwood <prestwoj@gmail.com>
Subject: [PATCH 1/5] band: correct oper class 136 starting frequency
Date: Thu, 24 Oct 2024 09:46:31 -0700
Message-Id: <20241024164635.176631-1-prestwoj@gmail.com>
X-Mailer: git-send-email 2.34.1
Precedence: bulk
X-Mailing-List: iwd@lists.linux.dev
List-Id: <iwd.lists.linux.dev>
List-Subscribe: <mailto:iwd+subscribe@lists.linux.dev>
List-Unsubscribe: <mailto:iwd+unsubscribe@lists.linux.dev>
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit

This should be 5925, not 5950
---
 src/band.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/band.c b/src/band.c
index 14daeb37..4e8eb9b0 100644
--- a/src/band.c
+++ b/src/band.c
@@ -896,7 +896,7 @@ static const struct operating_class_info e4_operating_classes[] = {
 	},
 	{
 		.operating_class = 136,
-		.starting_frequency = 5950,
+		.starting_frequency = 5925,
 		.channel_spacing = 20,
 		.center_frequencies = { 2 },
 	}
-- 
2.34.1


